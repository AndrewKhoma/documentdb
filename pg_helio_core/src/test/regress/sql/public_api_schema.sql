-- show all functions exported in helio_api.
\df helio_core.*

-- show all aggregates exported
\da+ helio_core.*

-- Access methods + Operator families
\dA *helio*

\dAc *helio*

\dAf *helio*

\dX *helio*

-- This is last (Tables/indexes)
\d helio_core.*
