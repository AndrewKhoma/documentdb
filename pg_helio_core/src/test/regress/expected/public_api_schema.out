-- show all functions exported in helio_api.
\df helio_core.*
                                                  List of functions
   Schema   |           Name            |    Result data type     |            Argument data types             | Type 
------------+---------------------------+-------------------------+--------------------------------------------+------
 helio_core | bson_compare              | integer                 | helio_core.bson, helio_core.bson           | func
 helio_core | bson_equal                | boolean                 | helio_core.bson, helio_core.bson           | func
 helio_core | bson_from_bytea           | helio_core.bson         | bytea                                      | func
 helio_core | bson_get_value            | helio_core.bson         | helio_core.bson, text                      | func
 helio_core | bson_get_value_text       | text                    | helio_core.bson, text                      | func
 helio_core | bson_gt                   | boolean                 | helio_core.bson, helio_core.bson           | func
 helio_core | bson_gte                  | boolean                 | helio_core.bson, helio_core.bson           | func
 helio_core | bson_hash_int4            | integer                 | helio_core.bson                            | func
 helio_core | bson_hash_int8            | bigint                  | helio_core.bson, bigint                    | func
 helio_core | bson_hex_to_bson          | helio_core.bson         | cstring                                    | func
 helio_core | bson_in                   | helio_core.bson         | cstring                                    | func
 helio_core | bson_json_to_bson         | helio_core.bson         | text                                       | func
 helio_core | bson_lt                   | boolean                 | helio_core.bson, helio_core.bson           | func
 helio_core | bson_lte                  | boolean                 | helio_core.bson, helio_core.bson           | func
 helio_core | bson_not_equal            | boolean                 | helio_core.bson, helio_core.bson           | func
 helio_core | bson_object_keys          | SETOF text              | helio_core.bson                            | func
 helio_core | bson_operator_selectivity | double precision        | internal, oid, internal, integer           | func
 helio_core | bson_out                  | cstring                 | helio_core.bson                            | func
 helio_core | bson_recv                 | helio_core.bson         | internal                                   | func
 helio_core | bson_repath_and_build     | helio_core.bson         | VARIADIC "any"                             | func
 helio_core | bson_send                 | bytea                   | helio_core.bson                            | func
 helio_core | bson_to_bson_hex          | cstring                 | helio_core.bson                            | func
 helio_core | bson_to_bsonsequence      | helio_core.bsonsequence | helio_core.bson                            | func
 helio_core | bson_to_bytea             | bytea                   | helio_core.bson                            | func
 helio_core | bson_to_json_string       | cstring                 | helio_core.bson                            | func
 helio_core | bson_typanalyze           | boolean                 | internal                                   | func
 helio_core | bson_unique_index_equal   | boolean                 | helio_core.bson, helio_core.bson           | func
 helio_core | bsonquery_compare         | integer                 | helio_core.bson, helio_core.bsonquery      | func
 helio_core | bsonquery_compare         | integer                 | helio_core.bsonquery, helio_core.bsonquery | func
 helio_core | bsonquery_equal           | boolean                 | helio_core.bsonquery, helio_core.bsonquery | func
 helio_core | bsonquery_gt              | boolean                 | helio_core.bsonquery, helio_core.bsonquery | func
 helio_core | bsonquery_gte             | boolean                 | helio_core.bsonquery, helio_core.bsonquery | func
 helio_core | bsonquery_in              | helio_core.bsonquery    | cstring                                    | func
 helio_core | bsonquery_lt              | boolean                 | helio_core.bsonquery, helio_core.bsonquery | func
 helio_core | bsonquery_lte             | boolean                 | helio_core.bsonquery, helio_core.bsonquery | func
 helio_core | bsonquery_not_equal       | boolean                 | helio_core.bsonquery, helio_core.bsonquery | func
 helio_core | bsonquery_out             | cstring                 | helio_core.bsonquery                       | func
 helio_core | bsonquery_recv            | helio_core.bsonquery    | internal                                   | func
 helio_core | bsonquery_send            | bytea                   | helio_core.bsonquery                       | func
 helio_core | bsonsequence_from_bytea   | helio_core.bsonsequence | bytea                                      | func
 helio_core | bsonsequence_get_bson     | SETOF helio_core.bson   | helio_core.bsonsequence                    | func
 helio_core | bsonsequence_in           | helio_core.bsonsequence | cstring                                    | func
 helio_core | bsonsequence_out          | cstring                 | helio_core.bsonsequence                    | func
 helio_core | bsonsequence_recv         | helio_core.bsonsequence | internal                                   | func
 helio_core | bsonsequence_send         | bytea                   | helio_core.bsonsequence                    | func
 helio_core | bsonsequence_to_bytea     | bytea                   | helio_core.bsonsequence                    | func
 helio_core | row_get_bson              | helio_core.bson         | record                                     | func
(47 rows)

-- show all aggregates exported
\da+ helio_core.*
                     List of aggregate functions
 Schema | Name | Result data type | Argument data types | Description 
--------+------+------------------+---------------------+-------------
(0 rows)

-- Access methods + Operator families
\dA *helio*
List of access methods
 Name | Type 
------+------
(0 rows)

\dAc *helio*
                  List of operator classes
 AM | Input type | Storage type | Operator class | Default? 
----+------------+--------------+----------------+----------
(0 rows)

\dAf *helio*
        List of operator families
 AM | Operator family | Applicable types 
----+-----------------+------------------
(0 rows)

\dX *helio*
                 List of extended statistics
 Schema | Name | Definition | Ndistinct | Dependencies | MCV 
--------+------+------------+-----------+--------------+-----
(0 rows)

-- This is last (Tables/indexes)
\d helio_core.*
